runtime/development/src/lib.rs

```diff
@@ -856,6 +856,18 @@ impl pallet_pools::Config for Runtime {
 	type PoolCreateOrigin = EnsureSigned<AccountId>;
 	type WeightInfo = pallet_pools::SubstrateWeight<Runtime>;
 	type TrancheWeight = TrancheWeight;
+	type PoolCurrency = PoolCurrencyChecker;
+}
+
+struct PoolCurrencyChecker;
+impl PoolCurrency for PoolCurrencyChecker {
+	type CurrencyId = CurrencyId;
+	fn base(id: Self::CurrencyId) -> bool {
+		match id {
+			CurrencyId::Tranche(_, _) | CurrencyId::Native => false,
```

Comment:
Here it's fine, but we should remember to re-enable `Native` as supported on `altair`, since the first pool on Altair will launch with AIR as currency.